src/mongo/db/pipeline/document_source_test.cpp

```diff
@@ -3371,6 +3371,33 @@ class OutputSort : public Mock::Base {
 
         ASSERT_EQUALS(outputSort.count(BSON("a" << 1)), 1U);
         ASSERT_EQUALS(outputSort.count(BSON("d.e" << 1)), 0U);
```

Comment:
I think you can delete 3373-3374, since they are redundant with the assertions on lines 3372 and line 3375.